ebean-core/src/main/java/io/ebeaninternal/server/deploy/AssocOneHelpEmbedded.java

```diff
@@ -32,7 +32,7 @@ Object readSet(DataReader reader, EntityBean bean) throws SQLException {
 
   @Override
   Object read(DataReader reader) throws SQLException {
-    EntityBean embeddedBean = property.targetDescriptor.createEntityBean();
+    EntityBean embeddedBean = property.targetDescriptor.createEntityBean2(true);
```

Comment:
This look like it includes non-id embedded beans

Reply:
Hmm, yes. This is should be done only for ID beans